core/tee/tee_fs_private.h

```diff
@@ -43,11 +46,36 @@
 #define TEE_FS_READDIR   12
 #define TEE_FS_RMDIR     13
 #define TEE_FS_ACCESS    14
+#define TEE_FS_LINK      15
+
+#define FILE_BLOCK_SHIFT	4
+
+#define FILE_BLOCK_SIZE		(1 << FILE_BLOCK_SHIFT)
+
+#define NUM_BLOCKS_PER_FILE	1024
+
+#define MAX_FILE_SIZE	(FILE_BLOCK_SIZE * NUM_BLOCKS_PER_FILE)
```

Comment:
So 16kB is the maximum allowed file size? Or can you consecutively append data to a single file? The question I'm asking is what to do if we want to write files bigger than 16kB? 

And by the way I cannot see that this define is used and therefore I guess this should be removed?